### PR TITLE
Update of FRENCH translation for Ajax Shoutbox extension

### PR DESCRIPTION
**Pull request by Galixte**
Updates for :
- acp_ajax_shoutbox.php
- ajax_shoutbox.php
- info_acp_ajaxshoutbox.php
- permissions_ajax_shoutbox.php

### CODE REVIEW

File: language/fr/acp_ajax_shoutbox.php [hunk 1]
```diff
@@ -10,8 +10,8 @@
  */
 
 /**
- * DO NOT CHANGE
- */
+* DO NOT CHANGE
+*/
 if (!defined('IN_PHPBB'))
```
Comment (paul999) [inline language/fr/acp_ajax_shoutbox.php:15]: Please undo these whitespace changes (This applies for the other files as well).

File: language/fr/permissions_ajax_shoutbox.php [hunk 2]
```diff
@@ -35,7 +35,7 @@
 // in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
 //
 // Some characters you may want to copy&paste:
-// ’ » “ ” …
+// ’ « » “ ” …
```
Comment (paul999) [inline language/fr/permissions_ajax_shoutbox.php:38]: Please undo these changes as well.

Comment (Skouat) [inline language/fr/permissions_ajax_shoutbox.php:38]: Hi paul999,

For french people, `« »` is same as `“ ”`
And the chars `«` is not present on our French keyboard. That's why we added this chars on this line.

So... why request to remove it ?
The line before says `
 // Some characters you may want to copy&paste:` and the copy/paste is the only way to use this chars easily.

Regards.

Comment (paul999) [inline language/fr/permissions_ajax_shoutbox.php:38]: Ok, it is fine in that case. Just fix the whitespace for all files in that case.

**Comment by Galixte** [comment]
Hi,

it’s done. Sacré polo, merci Skouat.

EDIT: pff the errors don’t result from whitespaces changes, there are Travis errors like that https://travis-ci.org/paul999/ajax-shoutbox-ext/jobs/62961276

**Comment by paul999** [comment]
Travis is fine, I still need to fix it for the stable branch,
